.lib/git-fleximod/License

```diff
@@ -0,0 +1,20 @@
+Copyright 2024 National Center for Atmospheric Sciences (NCAR)
```

Comment:
Oh boy, I think this should be "NSF National Center for Atmospheric Sciences"

Reply:
Fixed